src/components/Switch/__stories__/index.stories.tsx

```diff
@@ -5,6 +5,53 @@ import { Box, Icon } from '../..'
 import ControlValue from '../../../__stories__/components/ControlValue'
 
 export default {
+  argTypes: {
+    checked: {
+      control: { type: 'boolean' },
+      defaultValue: false,
+      description: 'checked',
+      name: 'checked',
+      table: {
+        defaultValue: { summary: false },
+        type: { summary: 'boolean' },
+      },
+      type: { name: 'boolean' },
+    },
+    disabled: {
+      control: { type: 'boolean' },
+      defaultValue: false,
+      description: 'disabled',
+      name: 'disabled',
+      table: {
+        defaultValue: { summary: false },
+        type: { summary: 'boolean' },
+      },
+      type: { name: 'boolean' },
+    },
+    labeled: {
+      control: { type: 'boolean' },
+      defaultValue: false,
+      description: 'labeled',
+      name: 'labeled',
+      table: {
+        defaultValue: { summary: false },
+        type: { summary: 'boolean' },
+      },
+      type: { name: 'boolean' },
+    },
```

Comment:
aren't controls automatically detected by storybook ?

Reply:
it's happen sometimes, on a lot of complexe typed component even. 

The only solution is to add argsType to spec and default args to add controls.

Reply:
That's terrible, it means each time we change proptypes we will need to change argsTypes of storybook too 😢 

There is no other way? I don't like much this solution

Reply:
The only way we can get it is too export a proper type which not extend an imported type (here HTMLAttributes<HTMLElement>